scala/websocket/package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "play-scala-websocket-example",
+  "name": "websocket",
```

Comment:
There are more npm files where the name needs to be changed:
```sh
$ grep -r --include='package*json' 'name'
java/jpa/package-lock.json:  "name": "play-java-jpa-example",
java/websocket/package-lock.json:  "name": "play-java-websocket-example",
play-scala-websocket-example/package-lock.json:  "name": "play-scala-websocket-example",
```

Reply:
Fixed ✅ 
I synced `name` in `package.json` with `build.sbt` and `settings.gradle[.kts]`